php/ThreatExchangeConfig.php

```diff
@@ -23,7 +23,7 @@ class ThreatExchangeConfig {
   const APP_ID = 0;
   const APP_SECRET = 'secret';
 
-  const FACEBOOK_SERVER = 'https://graph.facebook.com';
+  const FACEBOOK_SERVER = 'https://graph.facebook.com/v2.4';
```

Comment:
Let's keep this explicit. Just in case there are changes in future versions, I like working code to stay on the current version. Although probably not a factor here, it's a good precedent to set.